packages/neuron-ui/src/components/MultisigAddress/hooks.ts

```diff
@@ -68,8 +80,9 @@ export const useConfigManage = ({ walletId, isMainnet }: { walletId: string; isM
     })
   }, [setEntities, walletId])
   const updateConfig = useCallback(
-    (id: number) => (alias: string | undefined) => {
-      updateMultisigConfig({ id, alias: alias || '' }).then(res => {
+    (id: number) => (e: React.SyntheticEvent<any>) => {
+      const { value } = e.target as HTMLInputElement
```

Comment:
```suggestion
    (id: number) => (e: React.SyntheticEvent<unknown>) => {
      if (!(e.target instanceof HTMLInputElement)) return
      const { value } = e.target
```

Reply:
What's the difference between removing `any` and updating in `unknown`?

Reply:
> What's the difference between removing `any` and updating in `unknown`?

`any` and `as` can break the type system, while `unknown` is also a special case in the type system. It accepts any type, but when using or transferring the value to another type that is not `unknown` or `any`, it needs to be converted to a valid type first. This avoids breaking the type system.